_maps/map_files/PubbyStation/PubbyStation.dmm

```diff
@@ -23003,7 +22999,7 @@
 	pixel_x = -5;
 	pixel_y = 30
 	},
-/obj/machinery/iv_drip,
+/obj/machinery/computer/cloning,
```

Comment:
your changelog should prolly reference this since it's a bit big

Reply:
Wait I did this here?